### DISCUSSION TITLE
Brew stops working after terminal session is closed

### DISCUSSION THREAD

**Discussion by DatDerpyDude81** [Homebrew/discussions#3459] [Getting started] [answered]
I have installed Brew in MACOS Monterey with the following commands. 
`mkdir homebrew && curl -L https://github.com/Homebrew/brew/tarball/master | tar xz --strip 1 -C homebrew`

`eval "$(homebrew/bin/brew shellenv)"`

`brew update --force --quiet`

`chmod -R go-w "$(brew --prefix)/share/zsh"`

I have tested it after the install, and it works, however closing the terminal session and opening it again causes terminal to say that the command brew is not found after running it. I can still see the homebrew directory, so Im guessing its in the wrong place for terminal to run it.

**Comment by Rylan12** [comment, marked as answer]
Well, it's certainly recommended that you install in the usual spot using the [primary install method](https://github.com/Homebrew/install#install-homebrew-on-macos-or-linux), but that probably isn't your problem. You need to place the `eval "$(homebrew/bin/brew shellenv)"` line in your shell profile file (e.g. `~/.bash_profile`, `~/.profile`, etc). That way, it will be run everytime you start a new shell.

**Reply by DatDerpyDude81** [reply]
Thanks. That worked ^_^

**Reply by TheAppleUser1** [reply]
what does that mean

**Reply by Qulxis** [reply]
Needed to use eval `"$(/opt/homebrew/bin/brew shellenv)"` but depends on homebrew installation path.
